static/js/pages/instances/InstanceDetailPanel.tsx

```diff
@@ -0,0 +1,150 @@
+  const btnProps = {
+    instance: instance,
+    notify: notify,
+  };
+
+  return (
+    <div className="p-panel p-instance-detail-panel">
+      <div className="p-panel__header p-instance-detail-panel--header">
+        <div className="p-panel__title">Instance details</div>
+        <div className="p-panel__controls">
+          <Button
+            appearance="base"
+            className="u-no-margin--bottom"
+            hasIcon
+            onClick={onClose}
+          >
+            <i className="p-icon--close"></i>
+          </Button>
+        </div>
+      </div>
+      <div className="p-panel__content p-instance-detail-panel--content">
+        <List
+          inline
+          className="u-float-right"
+          items={[
+            <OpenTerminalBtn key="terminal" instance={instance} />,
+            <OpenVgaBtn key="vga" instance={instance} />,
+            <OpenInstanceDetailBtn
+              key="details"
+              instanceName={instance.name}
+            />,
+          ]}
+        />
+        <table className="u-table-layout--auto">
+          <tbody>
+            <tr>
+              <th>Name</th>
+              <td>
+                <a href={`/instances/${instance.name}`}>{instance.name}</a>
+              </td>
+            </tr>
+            <tr>
+              <th>Status</th>
+              <td>
+                <List
+                  className="u-sv-3"
+                  inline
+                  items={[
+                    <span key="status">{instance.status}</span>,
+                    instance.status === "Running" ? (
+                      <StopInstanceBtn key="stop" {...btnProps} />
+                    ) : (
+                      <StartInstanceBtn key="start" {...btnProps} />
+                    ),
+                  ]}
+                />
+              </td>
+            </tr>
+            <tr>
+              <th>Type</th>
+              <td>{instance.type}</td>
+            </tr>
+            <tr>
+              <th>Architecture</th>
+              <td>{instance.architecture}</td>
+            </tr>
+            <tr>
+              <th>Date created</th>
+              <td>{isoTimeToString(instance.created_at)}</td>
+            </tr>
+            <tr>
+              <th>Last used</th>
+              <td>{isoTimeToString(instance.last_used_at)}</td>
+            </tr>
+          </tbody>
+        </table>
+        <h5 className="u-no-margin--bottom">Profiles</h5>
+        <List
+          className="u-no-margin--bottom"
+          items={instance.profiles.map((name) => (
+            <a key={name} href={`/profiles/${name}`}>
+              {name}
+            </a>
+          ))}
+        />
+        <h5 className="u-no-margin--bottom">Networks</h5>
+        <List
+          className="u-no-margin--bottom"
+          items={Object.values(instance.expanded_devices)
+            .filter(isNicDevice)
+            .map((item) =>
+              item.name ? item.name + ": " + item.network : item.network
```

Comment:
I propose using a template string here, to make it more readable: ``${item.name}: ${item.network}``

Reply:
good idea.